docs/en/week10/lecture10.sbv

```diff
@@ -7496,253 +7497,253 @@ phenomenon at all so we did not really
 have to do any special tricks with
 
 1:42:45.810,1:42:52.260
-Bachner we were able to use batch no
+batch norm, we were able to use batch norm
 
 1:42:47.730,1:42:54.930
-masses okay and it's not necessary for
+as it is (?) [Ishan] Okay and it's not necessary for
 
 1:42:52.260,1:42:57.150
-all the contrast if networks to not be
+all the contrastive networks to not
 
 1:42:54.930,1:42:59.790
-using batch norm it's okay to have the
+use batch norm? it's okay to have the
 
 1:42:57.150,1:43:02.760
-batch norm here it's yeah it's ten I
+batch norm layer? [Ishan] Yeah, I
 
 1:42:59.790,1:43:04.590
-mean for example for Cynthia and so on
+mean for example for SimCLR and so on,
 
 1:43:02.760,1:43:06.510
-they try and move the sink batch norm
+they try and move to same batch norm
 
 1:43:04.590,1:43:08.820
-because if want to emulate a large batch
+because they want to emulate a large batch
 
 1:43:06.510,1:43:12.000
-size so you might have to do some tweaks
+size, so you might have to do some tweaks
 
 1:43:08.820,1:43:13.680
-in batch norm but basically if you you
+in batch norm, but basically you
 
 1:43:12.000,1:43:15.150
-cannot avoid it clearly because if you
+cannot avoid it really, because if you
 
 1:43:13.680,1:43:17.190
 completely remove batch norm then
 
 1:43:15.150,1:43:20.970
-training these like very deep networks
+training these very deep networks
 
 1:43:17.190,1:43:25.020
-is generally very hard anyway okay do
+is generally very hard anyway. [Student] Okay, do
 
 1:43:20.970,1:43:27.090
-you think that Paul paper works with the
+you think that PIRL paper works with the
 
 1:43:25.020,1:43:29.520
-Bashan omnious because it uses a memory
+batch norm layer (?) because it uses a memory
```

Comment:
"layers" instead of "layer". Why the (?) here? You heard it correctly.

Reply:
I was missing the S :smile: